net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/util/PriorityUtil.java

```diff
@@ -0,0 +1,62 @@
+package net.sourceforge.pmd.eclipse.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.sourceforge.pmd.RulePriority;
+import net.sourceforge.pmd.eclipse.plugin.UISettings;
+import net.sourceforge.pmd.eclipse.ui.views.PriorityFilter;
+
+/**
+ * Priority Util is a manager of the available priorities in the app. This Util
+ * will update if the Violations Overview Priority Filter updates which will
+ * cause the File markers, Tree Markers, and Violation Outline markers to
+ * update.
+ * 
+ * @author Phillip Krall
+ *
+ */
+public class PriorityUtil {
+
+	private static PriorityFilter priorityFilter = new PriorityFilter();
+	static {
+		UISettings.currentPriorities(true);
```

Comment:
I think, this call should not be needed.

Reply:
Yeah you are right about this. I was grabbing the values and caching before. I must have missed this with the clean up.